### PR TITLE
Convert snake_case from API to camelCase

### PR DESCRIPTION
**Pull request by kntsoriano**
## Description
It seems that we only need this from API calls. When rendering initial data via `react-bootloader`, the keys are converted automatically.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
     "dayjs": "^1.8.35",
     "global": "^4.4.0",
+    "humps": "^2.0.1",
```
Comment (ErikSchierboom) [inline package.json:14]: While using a library is nice, it does come at the cost of increased bundle/download size. As we're only doing a relatively simple string transformation, what do you think about just writing it ourselves? Or even better, just copy-pasting it from SO: https://stackoverflow.com/a/2970588/2071395 ;)

Comment (ErikSchierboom) [reply package.json:14]: The cost of the library is estimated at 1.5Kb: https://bundlephobia.com/result?p=humps@2.0.1

**Review by ErikSchierboom — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (kntsoriano) [reply package.json:14]: We will also need the transformation back to snake case when passing back the data to the API. Would those two functions now equal 1.5kb? If not, I'm happy to just copy paste our own :)

Comment (ErikSchierboom) [reply package.json:14]: Right. They might actually. Let's go with the library for now, and we can always revisit later.

**Review by ErikSchierboom — APPROVED** [review]

**Comment by iHiD** [comment]
Nice. Thanks 🙂

**Pull request merged**